SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -165,6 +166,12 @@ public SingularityRequest checkSingularityRequest(SingularityRequest request, Op
       checkBadRequest(!request.getScheduleType().isPresent(), "ScheduleType can only be set for scheduled requests");
     }
 
+    if (request.getScheduleTimeZone().isPresent()) {
+      if (!ArrayUtils.contains(TimeZone.getAvailableIDs(), request.getScheduleTimeZone().get())) {
+        badRequest("scheduleTimeZone %s was not a valid timezone code (e.g. 'US/Eastern' or 'GMT')", request.getScheduleTimeZone().get());
```

Comment:
We can just mention that, then -- something like "... does not map to a valid Java TimeZone object." We should also make sure all the verb tenses match, i believe the rest are present tense